scripts/migration/ensure_bookmark_existence.py

```diff
@@ -14,7 +14,7 @@
 
 def get_targets():
     logger.info('Acquiring targets...')
-    targets = [u for u in User.find() if Node.find(Q('is_bookmark_collection', 'eq', True) & Q('is_deleted', 'eq', False) & Q('contributors', 'eq', u._id)).count() == 0]
+    targets = [u for u in User.find(Q('merged_by', 'eq', None)) if Node.find(Q('is_bookmark_collection', 'eq', True) & Q('is_deleted', 'eq', False) & Q('creator', 'eq', u._id)).count() == 0]
```

Comment:
Would it make sense for `OSFUser.merge_user` to skip transferring the merged user's bookmark collection?